File: week-02/day-01/draw_pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

pyramid_level = int(input('Levels of the pyramid: '))

star = '*'
empty = ' '
for i in range(1, pyramid_level*2, 2):
    y = (pyramid_level * 2 - i // 2)
    print(y * empty + i * star)

File: week-07/day-01/draw_pyramid.js

'use strict';

let lineCount = 4;

// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//


let star = '*';
let empty = ' ';

for (let i = 1; i < lineCount * 2; i += 2){
    let y = lineCount * 2 - Math.floor(i / 2);
    console.log(empty.repeat(y) + star.repeat(i))
}